CONTRIBUTING.md

```diff
@@ -108,12 +104,12 @@ You'll probably also want to add your non-localhost domains to the `allowedHosts
 ## Finding something to work on
 
 If this is your first time here, check out the label
-[outreachy onboarding](https://github.com/firefox-devtools/profiler/issues?q=is%3Aopen+is%3Aissue+label%3A%22outreachy+onboarding%22+-label%3Aassigned).
+[good first issue](https://github.com/firefox-devtools/profiler/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+-label%3Aassigned).
```

Comment:
maybe we can switch to `no:assignee` in the future and remove the assigned label. (but first we need to look at all currently assigned issues and see if they have an assignee)

Reply:
Yeah, it would be good to switch to that one!